src/CalcManager/pch.h

```diff
@@ -3,27 +3,15 @@
 
 #pragma once
 
-#ifndef WIN32_LEAN_AND_MEAN
-#define WIN32_LEAN_AND_MEAN
-#endif
-
-// Windows headers define min/max macros.
-// Disable it for project code.
-#define NOMINMAX
-
+#include <algorithm>
+#include <array>
 #include <assert.h>
-#include <windows.h>
```

Comment:
So we've eliminated all our dependencies on windows.h? That's exciting!!

Reply:
On the top-level windows.h header, yes, and it is exciting!  We still have winerror.h, although #211 makes that available cross-platform.  It should be simple to take it a step further and eliminate winerror as well.

Reply:
`intsafe.h` is on the chopping block soon too: https://github.com/Microsoft/calculator/pull/398. That would leave `ppltasks.h` as the only Windows include here.

Reply:
#211 gets rid of ppltasks so we're almost there.

Reply:
Indeed, just noticed. :tada: